aula-1/src/main/java/org/acelera/saopaulo/App3.java

```diff
@@ -10,30 +10,25 @@ public static void main(String[] args) {
         for (int index = 1; index <= 1_000; index++) {
             numeros.add(index);
         }
-        for (Integer numero : numeros) {
-            System.out.println(numero);
-        }
+        
+        numeros.forEach(System.out::println);
 
         List<Integer> pares = new ArrayList<>();
         List<Integer> impares = new ArrayList<>();
+        
+        numeros.forEach(x -> {
```

Comment:
:question: qual das duas formas vc acha mais legível? com um for ou sem o for?

Reply:
Acredito que sem o for fica mais legível e menos verboso.
Fica assim : 
Para cada número na List de numeros, verifique se ele é par ou ímpar.